packages/password-field/src/vaadin-password-field.js

```diff
@@ -170,7 +170,6 @@ export class PasswordField extends SlotStylesMixin(TextField) {
 
     if (this._revealNode) {
       this.__updateAriaLabel(this.i18n);
```

Comment:
**nit:** Maybe this call is unnecessary as long as there is an i18n property observer that updates the aria label at initialization.

Reply:
No, it's there by design (if you try to remove it, two tests will fail).